libs/ledger-live-common/src/hw/index.test.ts

```diff
@@ -0,0 +1,97 @@
+import { aTransportBuilder } from "@ledgerhq/hw-transport-mocker";
+import { registerTransportModule, open } from ".";
+import { CantOpenDevice, TransportError } from "@ledgerhq/errors";
+
+jest.useFakeTimers();
+
+describe("open", () => {
+  describe("When no transport working with the device has been registered", () => {
+    it("should return the associated opened Transport instance", async () => {
```

Comment:
wrong description here, should be "should throw a CantOpenDevice error"

Reply:
oupsi